src/ipyforcegraph/behaviors/dag_mode.py

```diff
@@ -0,0 +1,44 @@
+"""Forces for ``ipyforcegraph``.
+
+Using documentation from:
+
+- `force-graph Force Engine Configuration https://github.com/vasturiano/force-graph#force-engine-d3-force-configuration`_
+"""
+
+# Copyright (c) 2023 ipyforcegraph contributors.
+# Distributed under the terms of the Modified BSD License.
+
+import enum
+
+import ipywidgets as W
+import traitlets as T
+
+from ._base import Behavior
+
+
+class DAGMode(enum.Enum):
+    off = None
+    top_down = "td"
+    button_up = "bu"
+    left_right = "lr"
+    right_left = "rl"
+    radial_out = "radialout"
+    radial_in = "radialin"
+
+
+@W.register
+class DAGBehavior(Behavior):
```

Comment:
Should this inherit from `BaseD3Force`?  The `active` property smells a lot like what's already there...

If so, we could just put the `DAGBehavior` in `behaviors/forces.py`

Reply:
Then you could conceivably put in inside the `GraphForces.forces` dictionary. Most of the DAG behavior isn't a force in the same sense as the other pure D3 forces. But could change the frontend implementation to pull out the dag behavioral model if we this it would be clearer to treat the DAGs like a force.